widgets/common/src/filters/RuleOperatorDropdown.tsx

```diff
@@ -1,28 +1,52 @@
 import { camelCase } from 'lodash';
-import type { FunctionComponent } from 'react';
+import { FunctionComponent, useEffect, useState } from 'react';
 import type { DropdownItemProps } from 'semantic-ui-react/dist/commonjs/modules/Dropdown/DropdownItem';
+
 import { i18nPrefix } from './consts';
+import { FilterRuleRowType, LabelsFilterRuleOperators, AttributesFilterRuleOperators } from './types';
+import type { FilterRuleOperator } from './types';
+
+interface RuleOperatorDropdownProps {
+    ruleType: FilterRuleRowType;
+    onChange: (value: FilterRuleOperator) => void;
+    value: FilterRuleOperator;
+}
+
+function getDropdownOptions(operators: string[]) {
+    const { i18n } = Stage;
+    return operators.map(
+        (operator): DropdownItemProps => ({
+            text: i18n.t(`${i18nPrefix}.operatorsLabels.${camelCase(operator as string)}`),
```

Comment:
Optional:

The type assertion here seems unnecessary to me since `operator` is already a `string` :slightly_smiling_face: 

```suggestion
            text: i18n.t(`${i18nPrefix}.operatorsLabels.${camelCase(operator)}`),
```

Reply:
True. Leftover from my original take to iterate over type values. I'll change that.